app/controllers/admin/physical_tests_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::PhysicalTestsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
You need another before action that checks wether the user is an admin and redirects to Member dashboard if not (check the Admin::DashboardController for reference on the setup only, I just noticed that that before_action has no redirect and it should).

Reply:
Also, I suggest adding the dashboard layout to this controller as well.